cypress/support/testHelper.js

```diff
@@ -62,6 +62,53 @@ export const figureVisibility = figure => {
   figure.should('to.have.descendants', 'img');
 };
 
+export const metaDataTags = (metaDataTag, content) => {
```

Comment:
I think we should discuss this and refactor the file at a later date. This PR is to do with meta data rather than refactoring the Test Helper file.

I agree on the name change and will rename it something more specific.